src/pageLayouts/ilite/Team/team.import.less

```diff
@@ -11,6 +11,8 @@
 
   p {
     padding: 10px 0;
+    max-width: 903px;
```

Comment:
@jbiser361 I recommend going with 880px here to line it up with the tab buttons above instead of the line

Reply:
Also increase the padding: 20px 0;